sdk/textanalytics/ai-text-analytics/package.json

```diff
@@ -47,18 +47,18 @@
     "execute:samples": "npm run build:samples && npm run execute:js-samples && npm run execute:ts-samples",
     "extract-api": "tsc -p . && api-extractor run --local",
     "format": "prettier --write --config ../../.prettierrc.json \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
-    "integration-test:browser": "echo skipped",
-    "integration-test:node": "echo skipped",
+    "integration-test:browser": "cross-env TEST_MODE=live npm run unit-test:browser",
+    "integration-test:node": "cross-env TEST_MODE=live npm run unit-test:node",
```

Comment:
I remember that @HarshaNalluru had different integration-test and unit-test scripts. I believe part of the reason was to make it easier to debug the tests.

Reply:
Yeah, just copy the ones from storage or keyvault for this.

Reply:
We don't want any references of TEST_MODE in the package.json